### DISCUSSION TITLE
About unmatched route

### DISCUSSION THREAD

**Discussion by AndresdoSantos** [expo/router#722] [Q&A]
Hey guys

I just really wanted to understand why the main page of the app is "unmatched".

I had a structure that was `app/index.tsx` but it always took me to the unmatched route.

I changed the content of the index to `[...unmatched].tsx` and "it worked" because the first page rendered is unmatched.

But, wasn't it supposed to be `index.tsx`?

**Comment by danilopolani** [comment]
I don't know if it may be the issue, but do you have a `_layout.tsx` page with a Stack/Tabs navigator to include the `index.tsx` page?

**Comment by budzHors7** [comment]
**Try this**

inside the [...unmatched].js, write a basic Not found Page, don't return unmatched component, return custom View>...</View plus with a link to redirect to / or index

**Reply by IamSAL** [reply]
Thanks! its quick hack.

**Reply by marklawlor** [reply]
This repo is for v2 and contains outdated information. You most likely want to use the `+not-found` route which was released in `v3`
